Makefile

```diff
@@ -672,14 +696,22 @@ npm-update: node-check | node_modules
 .PHONY: fomantic
 fomantic: $(FOMANTIC_DEST)
 
-$(FOMANTIC_DEST): $(FOMANTIC_CONFIGS) | node_modules
-	@if [ ! -d node_modules/fomantic-ui ]; then \
-		npm install --no-save --no-package-lock fomantic-ui@2.8.7; \
-	fi
-	rm -rf $(FOMANTIC_DEST_DIR)
-	cp -f web_src/fomantic/theme.config.less node_modules/fomantic-ui/src/theme.config
-	cp -rf web_src/fomantic/_site/* node_modules/fomantic-ui/src/_site/
-	npx gulp -f node_modules/fomantic-ui/gulpfile.js build
+$(FOMANTIC_WORK_DIR)/node_modules/fomantic-ui:
+	ln -sf ../../semantic.json $(FOMANTIC_WORK_DIR)
+	cd $(FOMANTIC_WORK_DIR); \
+		rm -rf node_modules && mkdir node_modules && \
+		npm install less@3 fomantic-ui --no-package-lock; \
```

Comment:
Is this `less@3` needed because main project uses `less@4`? Shouldn't that be a non-issue because fomantic has its own `node_modules` and depends on `less`?

Reply:
At least it won't work when `less@4` is installed. I agree specifying `less@3` explicitly with `fomantic-ui` is unnecessary.